package.json

```diff
@@ -20,8 +20,8 @@
     "ui-components"
   ],
   "scripts": {
-    "lint": "eslint src/",
-    "lint-demo": "eslint demo/src/",
+    "lint": "eslint --ext .ts --ext .tsx --ext .d.ts src/",
+    "lint-demo": "eslint --ext .ts --ext .tsx demo/src/",
```

Comment:
can we also add a build script
```
"build": "tsc",
```

Reply:
Sure, we can do it. @sshivananda . It will foster be for local typescript checks